doc/user-guide/genhelp.py

```diff
@@ -0,0 +1,165 @@
+import re
```

Comment:
No hashbang?
Also, zomg spaces? :-P (But sure, if you really really like that, just keep it out of the C files pls!)